packages/mutation-testing-metrics-scala/metrics/src/main/scala/mutationtesting/mutationTestResult.scala

```diff
@@ -8,6 +8,7 @@ final case class MutationTestReport(
     ),
     schemaVersion: String = "1",
     thresholds: Thresholds,
+    projectRoot: Option[String] = None,
```

Comment:
@hugo-vrijswijk lgtm, you?

Reply:
Made a small change. But otherwise looks good to me as well!